test/e2e/README.md

```diff
@@ -0,0 +1,18 @@
+# End to end tests
+
+* [Running e2e tests](../README.md#running-e2e-tests)
+
+## Adding end to end tests
+
+Knative Eventing Sources e2e tests [test the end to end functionality of the
+[Knative Eventing Sources](#requirements) to verify they work as expected.
+
+### Requirements
+
+The e2e tests are used to test whether events flow from Sources to Sinks as
```

Comment:
Is this implying there is a distinct test (suite?) for each source? Or is it only for KubernetesSources (which implicitly test ContainerSources)?

Reply:
I think this text is aspirational for when we have multiple tests. Currently there's only one test for KubernetesEventSource.